test/unit/component/example.nuxt.spec.ts

```diff
@@ -1,8 +1,10 @@
-import {it, expect} from "vitest"
-import {mountSuspended} from "vitest-environment-nuxt/utils"
+import {it, expect, describe} from "vitest"
+import {mountSuspended} from "@nuxt/test-utils/runtime"
 import {Example} from "#components"
 
-it("can mount some component", async () => {
-  const component = await mountSuspended(Example)
-  expect(component.html()).toMatchSnapshot("<p>Example</p>")
-})
\ No newline at end of file
+describe("Example Component", () => {
```

Comment:
Perhaps a test for the store could be added, but this is just an example file, so I'm unsure